### ISSUE TITLE
[6.7.4][PHP 5.2.11] Can't use method return value in write context in b2_blog_folder/inc/tools/views/_email_smtp.form.php on line 88

### ISSUE THREAD

**Issue by hhren1984** [b2evolution/b2evolution#28]
the error line is 

`if( $Settings->get( 'smtp_server_security' ) == 'none' || empty( $Settings->get( 'smtp_server_security' ) ) )`

I check the error detail info, it said:
> Note: empty() only checks variables as anything else will result in a parse error. In other words, the following will not work: empty(trim($name)).

so change it like this will be OK:
```
$smtp_server_security = $Settings->get( 'smtp_server_security' );
	if( $Settings->get( 'smtp_server_security' ) == 'none' || empty( $smtp_server_security ) )
	{
```

**Comment by keithbowes** [comment]
Why not?
`$smtp_server_security = $Settings->get( 'smtp_server_security' );
    if( $smtp_server_security == 'none' || empty( $smtp_server_security ) )
    {`

That would save from two different calls to get()

**Comment by hhren1984** [comment]
Yes, you are right, I neglect this.

**Comment by fplanque** [comment]
This has been fixed in 6.7.5.
